config/example-extras.cfg

```diff
@@ -58,6 +58,15 @@
 #pin_move_time: 0.200
 #   The amount of time (in seconds) that it takes the BLTouch pin to
 #   move up or down. The default is 0.200 seconds.
```

Comment:
I've no comment on this.  If the bulk of users with a genuine bltouch find this is needed then it can be updated.

Reply:
It's not only for genuine Bltouch owners, at least the Geeetech Bltouch clones need a similar time to detect errors. 

I don't think many users will report that they need a bigger move time, since the only thing that happens is that it fails to detect errors in time, for example that you are too close to the bed, and results in nozzle crashes. But they won't necessarily associate that with the configuration being wrong.

A slower value slows things down, especially the bed leveling, but I think the defaults should be for safety, where the users can choose to optimize the speed if they need to.